native/src/fairseq2n/data/finitude_type.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
I suggest putting this enum in data_source.h. No need to have a separate file just for this enum.